### DISCUSSION TITLE
Change size columns in q-table

### DISCUSSION THREAD

**Discussion by roylans** [quasarframework/quasar#17457] [General - Components / Directives / etc]
Is it possible to adjust the column width of a q-table directly through the user interface?

Thanks

**Comment by stefanvanherwijnen** [comment]
Set the `style` `width` of the column :

https://quasar.dev/vue-components/table/#defining-the-columns

Make it a computed property linked to an input to get what you want.

**Reply by fairking** [reply]
Could you please explain more how to do that?
I am particularly interesting in the following feature:

![screen_20240902_1224_647](https://github.com/user-attachments/assets/912d09fb-631a-4e0a-b462-34a68f34c264)

**Reply by stefanvanherwijnen** [reply]
That's not possible out of the box. I think it's possible by using custom slots, but that's too much work to show you exactly how to.

**Reply by fairking** [reply]
ok, thanks for your honest response.
